src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -146,7 +146,7 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
 
             const blockSettingsTag = schemaEntry.tags.find((tag) => tag.name === SETTINGS_TAG);
 
-            if (blockSettingsTag && schemaEntry.visible !== false) {
```

Comment:
This allowed to hide icons when the field for which the icon should be displayed was invisible. This is not so easy to rebuild anymore, because we only have the `visibleCondition` here. Therefore I just removed that check, I guess that wasn't really used anyway.